README.md

```diff
@@ -82,16 +161,167 @@ default.
 
 E.g. to build with the current system's config file run:
 
-    $ skt.py --workdir skt-workdir -vv \
+    $ skt.py --rc skt-rc --state --workdir skt-workdir -vv \
              build -c /boot/config-`uname -r`
 
-All the above commands use the `-vv` option to increase verbosity of the
-command's output, so it's easier to debug problems. Remove the option for
-quiter, shorter output.
-
-You can make skt output junit-compatible results by adding a `--junit
-<JUNIT_DIR>` option to any of the above commands. The results will be written
-to the `<JUNIT_DIR>` directory.
+### Publish
+
+To "publish" the resulting build using the simple "cp" (copy) publisher run:
+
+    $ skt.py --rc <SKTRC> --state --workdir <WORKDIR> -vv \
+             publish -p cp <DIRECTORY> <URL_PREFIX>
+
+Here `<DIRECTORY>` would be the location for the copied build artifacts, and
+`URL_PREFIX` would be the string to add to prepend the filenames with
+(together with a slash `/`) to construct the URLs the files will be reachable
+at. The resulting URLs will be passed to other commands, such as `run`, via
+the saved state in the configuration file.
```

Comment:
This paragraph is a little bit unclear surely because the command interface is tricky. If I understand correctly, you can use the simple "cp" only if the beaker-server is running on the same host that skt?